pilot/cmd/pilot-agent/options/options.go

```diff
@@ -152,4 +152,7 @@ var (
 	exitOnZeroActiveConnectionsEnv = env.Register("EXIT_ON_ZERO_ACTIVE_CONNECTIONS",
 		false,
 		"When set to true, terminates proxy when number of active connections become zero during draining").Get()
+
+	disableAgentSdsEnv = env.Register("DISABLE_AGENT_SDS", false,
```

Comment:
Should this be put in MeshConfig instead?

Reply:
How about `ENABLE_CUSTOM_SDS` as we are not really disabling sds